packages/schemas/defs.json

```diff
@@ -58,6 +58,41 @@
         "meteo/meteo--detail"
       ]
     },
+    "UIRepresentationPropertyConfiguration": {
+      "description": "Extend ui configuration field with pure UserInterface representation configuration (exemple specify placeholder from standard component, use slider component to set number instead simple number input, use color picker component to set string or yet use external widget).",
+      "type": "object",
+      "additionalProperties": false,
+      "minProperties": 1,
+      "properties": {
+        "widget": {
+          "description": "Specify widget to use. Widget is an external component hold by the current BlockProvider",
```

Comment:
Je reformulerais en ` Widget is similar to a component except it is owned by a BlockProvider and is not part of the standard components available in the platform.`

C'est se projeter un peu dans le futur mais je trouve que ca met mieux en valeur la différence entre component et widget.
Qu'en pensez-vous ? @FGRibreau

Reply:
Ca me parait mieux effectivement.
Je vais aussi pousser le terme Widget dans le Lexicon pour ouvrir la discussion